api4/team.go

```diff
@@ -25,6 +25,8 @@ func InitTeam() {
 	BaseRoutes.TeamByName.Handle("", ApiSessionRequired(getTeamByName)).Methods("GET")
 	BaseRoutes.TeamMember.Handle("", ApiSessionRequired(getTeamMember)).Methods("GET")
 	BaseRoutes.TeamMember.Handle("/roles", ApiSessionRequired(updateTeamMemberRoles)).Methods("PUT")
+
+	BaseRoutes.User.Handle("/teams/unread", ApiSessionRequired(getTeamsUnreadForUser)).Methods("GET")
```

Comment:
Minor but can you update this to use `BaseRoutes.TeamsForUser.Handler("/unread" ...`, it resolves to the same endpoint